example/MyFirstDrawing/MyFirstDrawing/ViewController.swift

```diff
@@ -32,6 +32,13 @@ class ViewController: UIViewController {
 }
 
 extension ViewController: DrawingDelegate {
+    func save(drawingImage: UIImage, drawingColor: String) {
+        print("Saved with drawingColor")
+    }
```

Comment:
Does those method to anything?

Reply:
The DrawingDelegate Protocol had these new methods. I just added them. Demo implementation needs to do nothing.